src/System.Net.Ping/tests/System.Net.Ping.Functional.Tests.builds

```diff
@@ -8,6 +8,11 @@
     <Project Include="FunctionalTests\System.Net.Ping.Functional.Tests.csproj">
       <TestTFMs>netcoreapp1.0;net46</TestTFMs>
       <OSGroup>Windows_NT</OSGroup>
+      <TargetGroup>netstandard1.3</TargetGroup>
+    </Project>
+    <Project Include="FunctionalTests\System.Net.Ping.Functional.Tests.csproj">
```

Comment:
Please put the default TargetGroup in the group at the top of the file see https://github.com/dotnet/corefx/pull/12312 for reference.